presidio-analyzer/conf/analyzer.yaml

```diff
@@ -0,0 +1,49 @@
+recognizer_registry:
+  recognizers: 
```

Comment:
How/can we have the engine load the predefined recognizers (call `RecognizerRegistry.load_predefined_recognizers()`)? How do we allow the user to provide their own recognizers vs. using the predefined ones or maybe a combination of the two?

Reply:
If we decide to have an opt-in approach, let's create a file with all existing recognizers (or at least all the global ones)

Reply:
So in the current implementation, if you don't provide recognizer_registry in the file, the predefined recognizers will be loaded

Reply:
Makes sense, how would you suggest to document this? Maybe adding a flag is more explicit?

Reply:
Currently all params are optional which is cool, but I get the point that it's unclear what recognizer is being chosen. 

Suggestion: 
recognizer_registry: predefined
and if nothing is given resolve to the default which is also predefined.

Reply:
Will that support the use case of predefined + new recognizers defined in the conf file?

Reply:
I have a suggestion: split to a list of predefined and a list of new. Anything on the predefined would look for the recognizer by name and the new would create new recognizers (like the existing example_recognizers file). We can initially put all the global or most common recognizers in the file to make sure everything is loaded. Wdyt?

Reply:
following up on that, at some point we decided to move all the predefined recognizers here right?